### PR TITLE
sigin& signup, filter ( in process)

### PR DESCRIPTION
**Pull request by bhtibrewal**
worked on sigin & signup, filter ( in process )

pages to check
- [ ] src/components/sidebar/Sidebar.jsx
- [ ] src/custom_hooks/useAxios.jsx
- [ ] src/pages/auth/sign_in/SignIn.jsx
- [ ] src/pages/auth/sign_up/SignUp.jsx
- [ ] src/pages/home_page/HomePage.jsx
- [ ] src/pages/product_page/ProductPage.jsx
- [ ] src/utils/getFilteredProductList.js

### CODE REVIEW

File: src/components/index.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { TextOverMediaCard } from './card/TextOverMediaCard';
 import { BasicCard } from './card/BasicCard';
-import {HorizontalCard} from './card/HorizontalCard'
+import { HorizontalCard } from './card/HorizontalCard'
 import { RatingPalleteIcon } from './rating/RatingPalleteIcon';
 import { ButtonPrimary, ButtonSecondary, OutlineButtonPrimary, OutlineButtonSecondary } from './buttons'
+import {Sidebar} from './sidebar/Sidebar';
```
Comment (yash9263) [inline src/components/index.js:6]: do export on the same line 
`export {HorizontalCard} from './card/HorizontalCard'`

**Review by yash9263 — APPROVED** [review]

**Comment by yash9263** [comment]
Everything looks good with nice use of custom hooks add the build folder in the .gitignore

File: src/custom_hooks/useAxios.jsx [hunk 2]
```diff
@@ -1,24 +1,38 @@
...
       try {
         const res = await axios(url, {
           method,
-          dataType,
-          body,
+          data: body,
           headers,
         });
-        if (res.status === 200) {
+        console.log({ res });
```
Comment (sri-rishi) [inline src/custom_hooks/useAxios.jsx:25]: Suggestion: `console.log` can be removed if it has no use

File: src/custom_hooks/useAxios.jsx [hunk 3]
```diff
@@ -1,24 +1,38 @@
...
+        if (res.status === 200 || res.status === 201) {
           setData(res.data[dataType]);
         }
-      } catch {
-        throw new Error("error");
+      } catch (e) {
+        console.log(e.response.data);
+        // throw new Error("error");
```
Comment (sri-rishi) [inline src/custom_hooks/useAxios.jsx:31]: Suggestion: remove comments if it's not usable before commits

File: src/utils/getFilteredProductList.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    return productsArr;
+}
+
+export const sortBy = (productsArr, filterState) => {
+    if (true)
+        return productsArr;
+}
+export const filterBy = (productsArr, filterState) => {
+    return productsArr;
+}
```
Comment (sri-rishi) [inline src/utils/getFilteredProductList.js:12]: Suggestion: Here `return` and  `{}` can be omitted

**Review by sri-rishi — COMMENTED** [review]
Great work, learnt new approach form  your code

**Pull request merged**